src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => {
+    const isMale = person.sex === 'm';
+
+    return century
+      ? isMale && Math.ceil(person.died / 100) === century
+      : isMale;
+  });
+
+  const calculateAge = men.map(person => person.died - person.born);
```

Comment:
```suggestion
  const menAge = men.map(person => person.died - person.born);
```
Don't name your variable like a function.  Function names usually start with the imperative verb